### DISCUSSION TITLE
Nextcloud Talk - phones cannot join to call (AIO 7.2.1)

### DISCUSSION THREAD

**Discussion by vawaver** [nextcloud/all-in-one#3472] [Questions] [answered]
I'd like to inquire about the issues with Nextcloud Talk in AIO. 
It seems impossible to initiate video calls from Android or iPhones. Here are the problems encountered when a call is initiated, involving participants with mobile phones:

1. Mobile users do not receive any notifications about an ongoing call.
2. Alternatively, if they attempt to join the call, it becomes unresponsive, leaving them waiting for others to join.
Tested devices: Samsung Galaxy S21+5G, iPhone 14 Pro, Huawei 11 Pro,  Huawei Mate 20.

I discovered that these issues arose after upgrading to version 7.00 and later. It's worth noting that I haven't made any changes to the settings on my AIO server, including port forwarding or any other configurations, since version AIO 4. I am certain that video calls were functional on my server in the past.

Could anyone please confirm if they have experienced similar issues?

**Comment by sol8712** [comment]
Hello! For #2, yes this has been broken the last update or 2 but a PR to fix this was pushed a few days ago and we are waiting on the next release.
**For me, i found i can connect if BOTH parties are OFF of the local network.** There is a settings fix for the issues and more info posted in the link below but i have not attempted it.
Note: the current 3.0 beta does not have the fixes.

I suggest keeping default settings, as much as possible, and holding out for the next release if you can.
Good luck!

https://github.com/nextcloud/all-in-one/discussions/3395


@szaimen [you] went above and beyond working with devs from an exTurnal project to get the issue fixed at the source and pushed here. Thank you for all the work you do your amazing!

**Reply by szaimen** [reply, marked as answer]
This is now fixed with v7.4.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Comment by vawaver** [comment]
> I suggest keeping default settings, as much as possible, and holding out for the next release if you can. Good luck!

That is exactly what I will do... wait for the next update and hope everything is fixed.
